Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -350,11 +350,52 @@ public class XPCServer {
         }
     }
     
-    private func handleError(_ error: Error,
-                             connection: xpc_connection_t,
-                             reply: inout xpc_object_t?) {
+    // MARK: Error handling
+    
+    /// Wrapper around an error handling closure to ensure there's only ever one error handler regardless of whether it's synchronous or asynchronous.
+    fileprivate enum ErrorHandler {
+        case none
+        case sync((XPCError) -> Void)
+        case async((XPCError) async -> Void)
+        
+        func handle(_ error: XPCError) {
+            switch self {
+                case .none:
+                    ()
```

Comment:
Doesn't this trigger an unused-value warning? This Void value isn't returned (implicit `return` only works for single-line expressions). Control flow falls out of here and falls to the end of the function normally.

Typically you would just use `break`, or in this case, just `return`.

Reply:
It does not trigger any warning. But `break` does indeed make more sense.